backend/sublet/serializers.py

```diff
@@ -51,54 +49,96 @@ def get_image_url(self, obj):
 
     class Meta:
         model = SubletImage
-        fields = ["image_url"]
+        fields = ["id", "image_url"]
 
 
 # complex sublet serializer for use in C/U/D + getting info about a singular sublet
 class SubletSerializer(serializers.ModelSerializer):
-    amenities = AmenitySerializer(many=True, required=False)
+    # amenities = AmenitySerializer(many=True, required=False)
+    # images = SubletImageURLSerializer(many=True, required=False)
+    amenities = serializers.PrimaryKeyRelatedField(
+        many=True, queryset=Amenity.objects.all(), required=False
+    )
 
     class Meta:
         model = Sublet
-        exclude = ["favorites"]
-        read_only_fields = ["id", "created_at", "subletter", "sublettees"]
+        read_only_fields = [
+            "id",
+            "created_at",
+            "subletter",
+            "sublettees",
+            # "images"
```

Comment:
nit: in addition to leaving this here, could you also leave a brief comment saying how `images` are created separately + are shown in the Read serializer?